docs/spherical-video-v2-rfc.md

```diff
@@ -13,13 +13,44 @@ GitHub.*
 ### MP4 (ISOBMFF)
 Spherical video metadata is stored in a new box, `sv3d`, defined in this RFC, in
 an MP4 (ISOBMFF) container. The metadata is applicable to individual video
-tracks in the container.
+tracks in the container. Since many spherical videos are also stereoscopic, this
+RFC also defines an additional optional box, `st3d`, to specify metadata specific
+to stereoscopic rendering.
 
 As the V2 specification stores its metadata in a different location, it is
 possible for a file to contain both the V1 and V2 metadata. If both V1 and V2
 metadata are contained they should contain semantically equivalent information,
 with V2 taking priority when they differ.
 
+#### Stereoscopic 3D Video Box (st3d)
+##### Definition
+Box Type: `st3d`  
+Container: Video Sample Description box (e.g. `avc1`, `mp4v`, `apcn`)  
+Mandatory: No  
+Quantity: Zero or one
+
+Stores additional information about stereoscopic rendering in this
+video track.
```

Comment:
Don't we want to avoid fallback / default value behavior? I'm concerned about users including the sv3d box but not the st3d hoping the fallback in monoscopic rendering.